Fixes

balnbibarbi opened the pull request:
The externalised config change is intended to be backward compatible. I hope I didn't miss anything?
The other two changes are trivial fixes.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/hygieia/api?pullRequest=213) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/hygieia/api?pullRequest=213) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/hygieia/api?pullRequest=213) it.</sub>

balnbibarbi commented:
> [![CLA assistant check](https://camo.githubusercontent.com/4c970efeed75dd01f6778bf7e33dec85e4171375cb69acb0927bfc9d437d78d9/68747470733a2f2f636c612d617373697374616e742e696f2f70756c6c2f62616467652f6e6f745f7369676e6564)](https://cla-assistant.io/hygieia/api?pullRequest=213)
> Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/hygieia/api?pullRequest=213) before we can accept your contribution.
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/hygieia/api?pullRequest=213) it.

I can't sign this on behalf of my employer. But these are very simple fixes. They are probably too short to be copyrightable.

balnbibarbi commented:
I'm informed that my employer has signed the CLA, so I think this is good to go now.

rvema commented:
Individual CLA is enforced and if the CLA checks fail we cannot accept the contribution.

rvema commented:
Closing PR due to CLA compliance. Please reopen once the individual CLA is signed.

balnbibarbi commented:
I've signed the individual CLA on behalf of my employer. But it still says "CLA not signed yet" above. I've tried clicking the "recheck" link. @rvema do you understand what's happening?
Also, I can't see a way to re-open this PR.

rvema commented:
@balnbibarbi we have not witnessed any issues with CLA however if your gitconfig is not configured properly say email is not associated with commit it may cause an issue. Please create a new PR with the same changes and try again.

balnbibarbi commented:
It's suddenly decided I've signed the CLA. I didn't do anything! Never mind.
Can you please re-open? Then I will fix the merge conflicts.

rvema commented:
@balnbibarbi can you update your branch?

balnbibarbi commented:
WhiteSource says I've introduced 7 new vulnerabilities in this branch. That's surprising, given that I didn't change any runtime dependencies. I just changed one compile-time dependency (the jacoco maven plugin). I think either I or WhiteSource have got confused somehow?

We can fix these vulnerabilities by upgrading to a new minor version of Spring Boot. The current 1.5.22 is the latest in the 1.5.x series, so we'd have to upgrade to 1.6.x at minimum. We could even upgrade to the latest 2.6.x.
In any case, upgrading Spring Boot is a somewhat disruptive change. If we decide to do it we should do it on a separate branch, not on this one.
